src/mame/heathkit/h89.cpp

```diff
@@ -221,4 +221,4 @@ ROM_END
 /* Driver */
 
 /*    YEAR  NAME  PARENT  COMPAT  MACHINE  INPUT  CLASS      INIT        COMPANY      FULLNAME        FLAGS */
-COMP( 1979, h89,  0,      0,      h89,     h89,   h89_state, empty_init, "Heath Company", "Heathkit H89", MACHINE_NOT_WORKING | MACHINE_NO_SOUND)
+COMP( 1979, h89,  0,      0,      h89,     h89,   h89_state, init, "Heath Company", "Heathkit H89", MACHINE_IS_INCOMPLETE | MACHINE_IMPERFECT_GRAPHICS | MACHINE_IMPERFECT_SOUND)
```

Comment:
`MACHINE_IS_INCOMPLETE` is for emulation of systems that were not completed (e.g. early prototypes, or machines that were not fully developed) as noted in the comment on its definition.  Is this the case for this system?

Also, driver initialisation functions are to be avoided apart from situations like needing to decrypt/unscramble ROMs.